.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: python
 python:
   - '2.7'
-before_install:
```

Comment:
Womp, looks like it's still an issue: https://travis-ci.org/simon-weber/gmusicapi/jobs/93860706. I'll revert just this commit.